corehq/apps/geospatial/es.py

```diff
@@ -136,16 +136,20 @@ def mid(lower, upper):
     return ceil(lower + (upper - lower) / 2)
 
 
-def case_query_for_missing_geopoint_val(domain, geo_case_property, case_type=None, size=None, offset=0):
+def case_query_for_missing_geopoint_val(
+        domain, geo_case_property, case_type=None, size=None, offset=0, should_sort=False
```

Comment:
Thanks for the change here.

nit: In order to give better control to the calling function, you could consider allowing to pass the field to sort by.

So, this would be `sort_by=None` and the calling function would pass the field they want to sort by.
I think that simplifies this further.

Reply:
I have refactored this a bit to use a `sort_by` field instead ([9089861](https://github.com/dimagi/commcare-hq/pull/35126/commits/9089861380c0f719b8da2c02449d5f5b3400702f)). The task and management command themselves still don't have control over this and we will still default to sorting by the `opened_on` field, however the util function will add this param depending on the doc count.